movies.py

```diff
@@ -29,22 +35,34 @@ def find_movie():
         if movie['title'] == get_title:
```

Comment:
removi for loop, replace with if else. 
```suggestion
    if get_title in movies['title']:
        print (f'{get_title} found.')
    else:
        print ('Not here')
```